src/User.cpp

```diff
@@ -0,0 +1,24 @@
+#include<User.hpp>
+
+void User::printUserdata()
+{
+    std::cout << name << " " << surname << " " << discord << " " << github << " " << firecode << std::endl;
+}
+
+void User::deleteUser()
+{
+    name = "";
+    surname = "";
+    discord = "";
+    github = "";
+    firecode = "";
+}
```

Comment:
This is not exactly deleting. It's just changing data. Deleating should be meant as removing the user from a collection.